src/utils/bbox.ts

```diff
@@ -7,9 +7,20 @@ export function getBoundingBoxByChildren(context: Context, svgnode: SvgNode): nu
   if (getAttribute(svgnode.element, context.styleSheets, 'display') === 'none') {
     return [0, 0, 0, 0]
   }
-  let boundingBox = [0, 0, 0, 0]
+  let boundingBox : number[] = []
   svgnode.children.forEach(child => {
     const nodeBox = child.getBoundingBox(context)
+    if ((nodeBox[0] === 0) && (nodeBox[1] === 0) && (nodeBox[2] === 0) && (nodeBox[3] === 0))
+       return
+    const transform = child.computeNodeTransform(context)
+    // TODO: take into account rotation matrix
+    nodeBox[0] = nodeBox[0] * transform.sx + transform.tx
+    nodeBox[1] = nodeBox[1] * transform.sy + transform.ty
+    nodeBox[2] = nodeBox[2] * transform.sx
+    nodeBox[3] = nodeBox[3] * transform.sy
```

Comment:
In think we should still transform all 4 corners and then calculate the encompassing axis-aligned rectangle.

Reply:
I will try